clusterloader2/testing/load/config.yaml

```diff
@@ -135,184 +132,30 @@ steps:
       - basename: daemonset-priorityclass
         objectTemplatePath: daemonset-priorityclass.yaml
 
-- name: Starting measurement for waiting for pods
-  measurements:
-  - Method: WaitForControlledPodsRunning
-    Instances:
-    - Identifier: WaitForRunningDeployments
-      Params:
-        apiVersion: apps/v1
-        kind: Deployment
-    - Identifier: WaitForRunningStatefulSets
-      Params:
-        apiVersion: apps/v1
-        kind: StatefulSet
-    - Identifier: WaitForRunningDaemonSets
-      Params:
-        apiVersion: apps/v1
-        kind: DaemonSet
-    - Identifier: WaitForRunningJobs
-      Params:
-        apiVersion: batch/v1
-        kind: Job
-    Params:
-      action: start
-      labelSelector: group = load
-      operationTimeout: 15m
+- module:
+    path: /modules/reconcile-objects.yaml
+    params:
+      actionName: "create"
       checkIfPodsAreUpdated: {{$CHECK_IF_PODS_ARE_UPDATED}}
```

Comment:
Can we pass this parameter as we do it for example for ENABLE_DNSTESTS?

Reply:
Done